MapboxNavigation/NavigationMapView.swift

```diff
@@ -270,8 +270,13 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         super.layoutSubviews()
         
         //If the map is in tracking mode, make sure we update the camera after the layout pass.
-        if (tracksUserCourse) {
-            updateCourseTracking(location: userLocationForCourseTracking, camera:self.camera, animated: false)
+        if (tracksUserCourse) {            
+            if #available(iOS 14.0, *) {
+                // Since layoutSubviews() is called too often on iOS 14.0, it leads to lags in `UserCourseView` animated updates.
+                // Workaround is to allow `UserCourseView` updates only on iOS versions lower than 14.0.
+            } else {
+                updateCourseTracking(location: userLocationForCourseTracking, camera:self.camera, animated: false)
```

Comment:
Just spitballing about alternatives that would allow us to keep updating the puck without it getting stuck on iOS 14:

* If the root cause is that kicking off a redundant view animation preempts any existing one, then `updateCourseTracking(location:camera:animated:)` should bail if `userLocationForCourseTracking` is unchanged.
* Perhaps for iOS 14 and above, NavigationMapView can keep a timestamp of the last time this method called `updateCourseTracking(location:camera:animated:)` and avoid calling it again within 1&nbsp;second (the typical interval between location updates).
* At one time, I think we had `updateCourseTracking(location:camera:animated:)` pass `UIView.AnimationOptions.beginFromCurrentState` into `UIView.animate(withDuration:delay:options:animations:completion:)`.

Reply:
Thanks for outlining possible alternatives. Since additional testing will require more time I think it'd be better to proceed with current implementation and consider other options in #2660.